llvm/test/CodeGen/X86/fold-int-pow2-with-fmul-or-fdiv.ll

```diff
@@ -1668,4 +1668,4 @@ define float @fdiv_pow_shl_cnt32_okay(i32 %cnt) nounwind {
   %conv = uitofp i32 %shl to float
   %mul = fdiv float 0x3a20000000000000, %conv
   ret float %mul
-}
+}
```

Comment:
end of file newline error

Reply:
fixed

Reply:
Still needs an extra empty line at the EOF